docs/ref/contrib/gis/geos.txt

```diff
@@ -325,7 +325,7 @@ Properties
     is simple if and only if it does not intersect itself (except at boundary
     points).  For example, a :class:`LineString` object is not simple if it
     intersects itself. Thus, :class:`LinearRing` and :class:`Polygon` objects
-    are always simple because they do cannot intersect themselves, by
+    are always simple because they cannot intersect themselves, by
```

Comment:
Nice catch 🏆

Doc line limit is 79 and lines should be stretched out to that limit meaning that the next word needs to be brought up.  Just FYI editors should be able to do this for your too.

Reply:
Done. Thank you!